### DISCUSSION TITLE
Sale Price via API

### DISCUSSION THREAD

**Discussion by kgulmer** [inventree/InvenTree#4097] [Q&A] [answered]
I am rolling our salable parts into a separate system and am trying to ascertain the sales price through the API using /api/part/?salable=true. 

The response includes pricing_max and pricing_min which seem to reflect (internal) BOM pricing, but I do not see the (external) Sale pricing. Am I missing something? If not, feature request to add sale_pricing_min and sale_pricing_max to the response from the API.

**Comment by matmair** [comment]
@kgulmer please use the provided issue template for feature requests.

**Comment by SchrodingersGat** [comment, marked as answer]
@kgulmer use `/api/part/<pk>/pricing/` for cached pricing information:

https://demo.inventree.org/api/part/10/pricing/
